### DISCUSSION TITLE
How can i reference chromium for puppeteer ?

### DISCUSSION THREAD

**Discussion by asotog** [vercel/vercel#5224] [Help]
I have a Scully.io application, but when it builds, hangs up, and waits forever:

![image](https://user-images.githubusercontent.com/813023/94030509-21514380-fd7b-11ea-9eac-ef498272c002.png)

based on documentation from Scully they suggest this

![image](https://user-images.githubusercontent.com/813023/94030275-ddf6d500-fd7a-11ea-97c7-24f4e4cf8ead.png)

but I'm not sure the proper values to pass by using env variables in Vercel,

could you please give me any clue about it?

**Comment by vniehues** [comment]
Did you find a solution to this problem? I'm facing it right now.
